lib/tilex_web/views/shared_view.ex

```diff
@@ -2,11 +2,20 @@ defmodule TilexWeb.SharedView do
   use TilexWeb, :view
 
   def display_date(post) do
-    Timex.format!(post.inserted_at, "%B %-e, %Y", :strftime)
+    post.inserted_at
+    |> datetime_in_zone()
+    |> Timex.format!("%B %-e, %Y", :strftime)
   end
 
   def rss_date(post) do
-    Timex.format!(post.inserted_at, "%a, %d %b %Y %H:%M:%S GMT", :strftime)
+    post.inserted_at
+    |> datetime_in_zone()
+    |> Timex.format!("%a, %d %b %Y %H:%M:%S GMT", :strftime)
```

Comment:
This is not correct, the result is going to be time in "America/Chicago" timezone pretending to be GMT. Consider this example:
```
iex(16)> :calendar.universal_time
{{2018, 1, 6}, {22, 2, 48}}                          
iex(17)> Timex.Timezone.convert(:calendar.universal_time, "America/Chicago") |> Timex.format!("%a, %d %b %Y %H:%M:%S GMT", :strftime)
"Sat, 06 Jan 2018 16:02:50 GMT"
```
RSS time should be GMT/UTC, so there is no need for conversion if your time in DB is all UTC.